cli/args/flags_net.rs

```diff
@@ -32,7 +33,11 @@ pub fn validator(host_and_port: &str) -> Result<String, String> {
   if Url::parse(&format!("internal://{host_and_port}")).is_ok()
     || host_and_port.parse::<IpAddr>().is_ok()
     || host_and_port.parse::<BarePort>().is_ok()
-    || NetDescriptor::parse(host_and_port).is_ok()
+    || NetDescriptor::parse_for_list(
+      host_and_port,
+      UnstableSubdomainWildcards::Enabled,
```

Comment:
Sweet. Thanks! This also makes it slightly easier to remove in the future because we'll just need to do a find all references for this type.